com.unity.shadergraph/Editor/Data/Util/FunctionRegistry.cs

```diff
@@ -15,10 +15,12 @@ class FunctionRegistry
         Dictionary<string, FunctionSource> m_Sources = new Dictionary<string, FunctionSource>();
         bool m_Validate = false;
         ShaderStringBuilder m_Builder;
+        IncludeCollection m_Includes;
```

Comment:
Easiest was to make the FunctionRegistry effectively a "FunctionAndNodeInclude" registry...